contracts/modules/commons/interfaces/IModuleCalls.sol

```diff
@@ -1,14 +1,17 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity 0.7.6;
+pragma solidity 0.8.17;
 pragma experimental ABIEncoderV2;
 
 
 interface IModuleCalls {
   // Events
-  event NonceChange(uint256 _space, uint256 _newNonce);
   event TxFailed(bytes32 _tx, bytes _reason);
   event TxExecuted(bytes32 _tx) anonymous;
 
+  // Errors
+  error NotEnoughGas(uint256 _requested, uint256 _available);
```

Comment:
Does it make sense to also add the transaction index here as well? We're doing it for `CallReverted` and MultiCallUtils' `DelegateCallNotAllowed`.